spec/api_spec.rb

```diff
@@ -35,13 +42,16 @@
   end
 
   describe "GET /interviewers" do
-    let(:collection) { [double(value: interviewer1, key: '001'), double(value: interviewer2, key: '002')] }
-    let(:interviewer1) { {name: 'Jichao', email: 'jichao@thoughtworks.com', key: '001'} }
-    let(:interviewer2) { {name: 'Ouyang', email: 'ouayng@thoughtworks.com', key: '002'} }
+    let(:collection) { [double(value: interviewer1_db, key: '001'), double(value: interviewer2_db, key: '002')] }
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.